api/V1/Monitoring.ts

```diff
@@ -0,0 +1,150 @@
+/*
+cacophony-api: The Cacophony Project API server
+Copyright (C) 2021  The Cacophony Project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import middleware, { toIdArray, toDate }  from "../middleware";
+import auth from "../auth";
+import e, { Application } from "express";
+import {calculateMonitoringPageCriteria, MonitoringParams}  from "./monitoringPage";
+import {generateVisits}  from "./monitoringVisit";
+import responseUtil from "./responseUtil";
+import { query } from "express-validator/check";
+
+export default function (app: Application, baseUrl: string) {
+  const apiUrl = `${baseUrl}/monitoring`;
+
+    /**
+     * @api {get} /api/v1/monitoring/page Get visits page
+     * @apiName MonitoringPage
+     * @apiGroup Monitoring
+     * @apiDescription Get a page of monitoring visits.   First recording are sorted into visits and then the best-classification for this visit is calculated.
+     * Optionally you can also a specify an ai so you can compare the best classification with that given by the ai.   
+     * 
+     * @apiUse V1UserAuthorizationHeader
+     * @apiParam {number} devices  A list of ids of devices to include
+     * @apiParam {number} groups  A list of ids of groups to include
+     * @apiParam {date} from  Retreive visits after this date
+     * @apiParam {date} until Retreive visits starting on or before this date
+     * @apiParam {number} page  Page number to retrieve
+     * @apiParam {number} page Maximum numbers of visits to show on each page
```

Comment:
Default value (100).  Max value (100)

Reply:
While this woud be ideal I am worried that it will get out of date as we change things.   It should error if the the user tries to retrieve more than 100.   I believe errrors are more useful documentation than the api docs.  As they are the information when you need it, where you need it.